README.rst

```diff
@@ -165,8 +165,9 @@ We can now install the REANA command-line client, run the analysis and download
 
 .. code-block:: console
 
-    $ # install REANA client
-    $ mkvirtualenv reana-client
+    $ # create new virtual environment
+    $ virtualenv ~/.virtualenvs/myreana
+    $ source ~/.virtualenvs/myreana/bin/activate
     $ pip install reana-client
```

Comment:
`$ # install REANA client`  line missing before `pip install...` command

Reply:
I'm sorry, it's fixed now.